react/components/FooterLinkList.js

```diff
@@ -4,10 +4,16 @@ import React from 'react'
 import footerList from './footerList'
 import footer from '../footer.css'
 
-export const FooterLinkItem = ({ url, title }) => (
+const targetByType = {
+  self_tab: '_self', 
```

Comment:
I would keep it `_self` and `_blank`, so there's nothing new to learn in the API.

Reply:
I used new_tabe and self_tab thinking about the final user when selecting that options on Text Editor, I think that is more intuitive. That make any sense? Or in Text Editor the options values can be translated? @klzns @brunojdo